client-rpc/src/rpc/multisig_rpc.rs

```diff
@@ -81,6 +91,33 @@ impl<T> MultiSigRpc for MultiSigRpcImpl<T>
 where
     T: WalletClient + MultiSigWalletClient + 'static,
 {
+    fn create_address(
+        &self,
+        request: WalletRequest,
+        public_keys: Vec<String>,
+        self_public_key: String,
+        required_signatures: usize,
```

Comment:
should it do some check on `required_signatures` vs `public_keys.len()`?
Or is that handled downstream in `new_multisig_transfer_address` ?

Reply:
or seems it's handled: https://github.com/crypto-com/chain/pull/279/files#diff-95f634f70a8d297a68cbd93dc8696949R195 ?

Reply:
Yes, most of the errors is handled downstream and is tested, I have added one more test but it's just to make it more complete.

The only check I added in the wallet_client layer is the checking of self public key belongs to the wallet.

Reply:
https://github.com/calvinaco/chain/blob/feature/cro-313-create-multisig-address-rpc/client-core/src/service/root_hash_service.rs#L49

Checking here but the error needs more polishing in the future.